cmd/nmi/main.go

```diff
@@ -52,6 +53,10 @@ var (
 	kubeconfig                         = pflag.String("kubeconfig", "", "Path to the kube config")
 )
 
+// Delay nmi startup due to DNS not being available during first seconds of nmi process execution.
+// Bug tracks removal of this delay: https://o365exchange.visualstudio.com/O365%20Core/_workitems/edit/1739605
```

Comment:
@kunZooberg  can we not have the VSTS link in here. These are public GitHub, lets have internal links in here.